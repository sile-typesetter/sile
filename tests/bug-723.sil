\begin[papersize=a6]{document}
\nofolios
\neverindent
\begin[first-content-frame=ltrttb]{pagetemplate}
\frame[id=ltrttb,direction=LTR-TTB,top=top(content),left=left(content),right=right(content),height=10%ph,next=ltrbtt]
\frame[id=ltrbtt,direction=LTR-BTT,top=bottom(ltrttb),left=left(content),right=right(content),height=10%ph,next=rtlttb]
\frame[id=rtlttb,direction=RTL-TTB,top=bottom(ltrbtt),left=left(content),right=right(content),height=10%ph,next=rtlbtt]
\frame[id=rtlbtt,direction=RTL-BTT,top=bottom(rtlttb),left=left(content),right=right(content),height=10%ph,next=ttbltr]
\frame[id=ttbltr,direction=TTB-LTR,top=bottom(rtlbtt),left=left(content),width=10%ph,bottom=bottom(content),next=ttbrtl]
\frame[id=ttbrtl,direction=TTB-RTL,top=top(ttbltr),left=right(ttbltr),width=10%ph,bottom=bottom(ttbltr),next=bttltr]
\frame[id=bttltr,direction=BTT-LTR,top=top(ttbrtl),left=right(ttbrtl),width=10%ph,bottom=bottom(ttbrtl),next=bttrtl]
\frame[id=bttrtl,direction=BTT-RTL,top=top(bttltr),left=right(bttltr),width=10%ph,bottom=bottom(bttltr)]
\end{pagetemplate}
\begin[command=dots]{define}
\par
\hrule[width=1em,height=1em]
\hrule[width=1em,height=2em]
\par
\end{define}
\script[src=packages/rules]
\dots\framebreak
\dots\framebreak
\dots\framebreak
\dots\framebreak
\dots\framebreak
\dots\framebreak
\dots\framebreak
\dots
\end{document}
