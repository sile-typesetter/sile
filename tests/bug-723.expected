Set paper size 	297.6377985	419.5275636
Begin page
Draw line	14.8819	30.9764	10.0000	10.0000
Draw line	24.8819	20.9764	10.0000	20.0000
Draw line	14.8819	94.8819	10.0000	-10.0000
Draw line	24.8819	104.8819	10.0000	-20.0000
Draw line	282.7559	114.8819	-10.0000	10.0000
Draw line	272.7559	104.8819	-10.0000	20.0000
Draw line	282.7559	178.7874	-10.0000	-10.0000
Draw line	272.7559	188.7874	-10.0000	-20.0000
Draw line	24.8819	188.7874	10.0000	10.0000
Draw line	14.8819	198.7874	20.0000	10.0000
Draw line	88.7874	188.7874	-10.0000	10.0000
Draw line	98.7874	198.7874	-20.0000	10.0000
Draw line	108.7874	377.5748	10.0000	-10.0000
Draw line	98.7874	367.5748	20.0000	-10.0000
Draw line	172.6929	377.5748	-10.0000	-10.0000
Draw line	182.6929	367.5748	-20.0000	-10.0000
End page
Finish
